pkg/controller/operators/operator_controller.go

```diff
@@ -144,11 +150,19 @@ func (r *OperatorReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 	}
 
-	if err := r.Status().Update(ctx, operator.Operator); err != nil {
-		log.Error(err, "Could not update Operator status")
-		return ctrl.Result{}, err
+	if create {
+		if err := r.Create(context.Background(), operator); err != nil && !apierrors.IsAlreadyExists(err) {
+			r.log.Error(err, "Could not create Operator", "operator", name)
+		}
+	} else {
+		if err := r.Status().Update(ctx, operator.Operator); err != nil {
+			log.Error(err, "Could not update Operator status")
+			return ctrl.Result{}, err
+		}
 	}
 
+	r.setLastResourceVersion(req.NamespacedName, operator.Operator.ResourceVersion)
```

Comment:
are we sure that the Create and Status.Update() methods bump the ResourceVersion on the resource argument?

Reply:
Yes, I checked this.  That is why Create/Update take pointers to the resource so they can return the resource after the server has processed it.